### PR TITLE
Feature/orca 513 Improve logging in integration tests

### PR DESCRIPTION
**Pull request by rizbihassan**
## Summary of Changes

Please write a sentence or two summarizing the proposed change.

Addresses [ORCA-513: Improve logging in integration tests
](https://bugs.earthdata.nasa.gov/browse/ORCA-513)

## Changes

* added custom logger to categorize integration tests

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Ran the integration tests and verified tests are categorized.

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas

- [x ] My changes generate no new warnings
- [ x] My code has passed security scanning
    - [x ] Snyk
    - [x ] git-secrets

## What is the expected correct/added behavior?

Tests are logged as 
`[Ingest TestMultipleGranulesFilesExcluded]`, 
`[Ingest TestOverrideStorageClassHappyPath]`, 
`[Ingest TestProviderDoesNotExist]`, 
`[Ingest TestMultipleGranulesSameIdDifferentCollections]`, 
`[Ingest TestMultipleGranulesHappyPath]`
`[Ingest TestNoGranules"]`

### CODE REVIEW

File: integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py [hunk 1]
```diff
@@ -263,3 +265,26 @@ def test_multiple_granules_happy_path(self):
...
+#         logger.addHandler(syslog)
+#         adapter = CustomAdapter(logger,{'my_context': group_name})
+#         return adapter
+
+# logger = CustomAdapter.set_logger("Ingest test logger")
+# logger.setLevel(logging.INFO)
+
+# logger.info('The sky is so blue')
+# logger.error("test")
+# logger.critical("test critical")
```
Comment (AndrewDorn) [inline integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py:290]: Old test data

File: integration_test/workflow_tests/test_packages/catalog/test_provider_does_not_exist.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-import logging
 import time
 import uuid
 from unittest import TestCase
 
 import helpers
+from custom_logger import CustomLoggerAdapter
 
+#Set the logger
+logging = CustomLoggerAdapter.set_logger("Ingest TestProviderDoesNotExist")
```
Comment (AndrewDorn) [inline integration_test/workflow_tests/test_packages/catalog/test_provider_does_not_exist.py:10]: I might fiddle with the [__name__](https://www.tutorialspoint.com/how-do-i-find-the-current-module-name-in-python#:~:text=A%20module%20can%20find%20out,is%20running%20as%20a%20script.) stuff to see if we can fully automate the parameter for set_logger. Not needed for this card though.

File: integration_test/workflow_tests/custom_logger.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import logging
+
+
+class CustomLoggerAdapter(logging.LoggerAdapter):
+    # example taken from
+    # https://stackoverflow.com/questions/39467271/cant-get-this-custom-logging-adapter-example-to-work
+    def process(self, msg, kwargs):
+        my_context = kwargs.pop('my_context', self.extra['my_context'])
+        return '[%s] %s' % (my_context, msg), kwargs
```
Comment (AndrewDorn) [inline integration_test/workflow_tests/custom_logger.py:9]: I think you can simplify this to `return f"[{my_context}] msg", kwargs`
Check me on that.

**Review by AndrewDorn — COMMENTED** [review]
Looks good. Couple minor cleanup items.

File: integration_test/workflow_tests/test_packages/catalog/test_provider_does_not_exist.py (hunk 2 above)
Comment (rizbihassan) [reply integration_test/workflow_tests/test_packages/catalog/test_provider_does_not_exist.py:10]: Sounds good.

File: integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py (hunk 1 above)
Comment (rizbihassan) [reply integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py:290]: Oops removed!

File: integration_test/workflow_tests/custom_logger.py (hunk 3 above)
Comment (rizbihassan) [reply integration_test/workflow_tests/custom_logger.py:9]: Tried and looks like it does not show the logs properly. For example, I get this after running the test where it just shows as `msg`

```
[Ingest TestOverrideStorageClassHappyPath] msg
[Ingest TestOverrideStorageClassHappyPath] msg
[Ingest TestOverrideStorageClassHappyPath] msg
```

**Review by AndrewDorn — APPROVED** [review]

**Pull request merged**